jmeter-service/deploy/service.yaml

```diff
@@ -36,10 +36,22 @@ spec:
         ports:
         - containerPort: 8080
         env:
+        - name: DT_API_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: dynatrace
+              key: DT_API_TOKEN
+        - name: DT_TENANT
+          valueFrom:
+            secretKeyRef:
+              name: dynatrace
+              key: DT_TENANT
```

Comment:
This will fail if the secret does not exist. Please make it optional by adding `optional: true` (see https://stackoverflow.com/a/48209289 for an example)

Reply:
Thanks. Added that to my service.yaml